mantle/platform/api/aws/images.go

```diff
@@ -329,21 +329,28 @@ func (a *API) CreateImportRole(bucket string) error {
 	return nil
 }
 
-func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, description string, architecture string) (string, error) {
+func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, description string, architecture string, volumetype string, imdsv2Only bool, uploadX86BootMode string) (string, error) {
 	var awsArch string
+	var bootmode string
 	if architecture == "" {
 		architecture = runtime.GOARCH
 	}
 	switch architecture {
 	case "amd64", "x86_64":
 		awsArch = ec2.ArchitectureTypeX8664
+		bootmode = uploadX86BootMode
 	case "arm64", "aarch64":
 		awsArch = ec2.ArchitectureTypeArm64
+		bootmode = "uefi"
 	default:
 		return "", fmt.Errorf("unsupported ec2 architecture %q", architecture)
 	}
 
-	return a.createImage(&ec2.RegisterImageInput{
+	// default to gp2
```

Comment:
The comment says one thing but the code does another.

```suggestion
	// default to gp3
```

Reply:
ah good catch sorry about that, I ment to get back to looking at my early morning changes yesterday but the day got away from me. Will fix.